projects/igniteui-angular/src/lib/calendar/calendar.component.ts

```diff
@@ -94,44 +94,81 @@ export class CalendarHammerConfig extends HammerGestureConfig {
     templateUrl: 'calendar.component.html'
 })
 export class IgxCalendarComponent implements OnInit, ControlValueAccessor {
-
-    @HostBinding('attr.id')
+/**
+ * Sets/gets the `id` of the calendar.
+ * If not set, the `id` will have value `"igx-calendar-0"`.
+ * ```html
+ * <igx-calendar id = "my-first-calendar"></igx-calendar>
+ * ```
+ * ```typescript
+ * let calendarId =  this.calendar.id;
+ * ```
+ * @memberof IgxCalendarComponent
+ */
+@HostBinding('attr.id')
     @Input()
     public id = `igx-calendar-${NEXT_ID++}`;
     /**
-     * An `@Input` property indicating the start of the week.
-     * Defaults to Sunday.
+     * Gets the start of the week.
```

Comment:
Mention also the WEEKDAYS enumerator, this will get enum value representation of the week day.
Also "Gets the start day of the week" is more clear.